backend/src/main/java/hu/flowacademy/konyhatunder/model/Recipe.java

```diff
@@ -39,5 +39,7 @@
     private Image image;
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private Integer recommendations;
-
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @OneToMany(mappedBy ="recipe")
+    private List<Comment> comments;
```

Comment:
En lazy init-tel csinalnam, es nem kell a jsonos magia, mert dto-ban adnam at, sosem defaultkent minden recepthez, csak ha explicit request erkezik a commentekert, amiket jo lenne page-elni is ha jut ra idotok, ha a demo-hoz nem lesz sok komment mukodhet page-eles nelkul de jo kis extra feature lenne, egy lapozgathato comment szekcio.

Reply:
A receptnél default megjelenik az összes hozzászólás frontenden, nem kéri le küldön.

Reply:
Ezt ertem, es ezen lenne jo valtoztatni, mert amint teli spammalem 5000 5000-soros hozzaszolassal fejre allnak a szerverek.

Reply:
A komment csak 255 karaktert engedélyez. :D 
Később ha marad időnk, külön issue-ba megnézzük a page-inget.

Reply:
A DTO + lazy inites reszt se felejtsetek ki a dologbol ;)